README.md

```diff
@@ -1,24 +1,272 @@
-# Requirements
+Corva python-sdk is a framework for building stream, scheduled and task apps.
+
+## Contents
+
+- [Requirements](#requirements)
+- [Installation](#installation)
+- [App types](#app-types)
+- [Event](#event)
+- [Api](#api)
+- [Cache](#cache)
+- [Contributing](#contributing)
+
+## Requirements
+
 Python 3.8+
 
-# Local development
+- [Pydantic](https://github.com/samuelcolvin/pydantic)
+- [Redis](https://pypi.org/project/redis/)
+- [Requests](https://pypi.org/project/requests/)
+- [urllib3](https://pypi.org/project/urllib3/)
+
+## Installation
+
+```console
+$ pip install corva-sdk
+```
+
+## App types
+
+There are three app types, that you can build: `stream`, `scheduled` and `task`.<br>
+**Note**: it is recommended to use type hints like in examples below,
+so that editors and tools can give you better support.
+
+#### Stream
+
+```python
+from corva import Api, Cache, Corva, StreamEvent
+
+
+# 1 define a function with required parameters, that will be provided by sdk
+def stream_app(event: StreamEvent, api: Api, cache: Cache):
+    """Main logic function"""
+    pass
```

Comment:
let's do something here. "Hello World" code at least

Reply:
Agreed. That would be nice.

Reply:
maybe a docstring following this format would explain things to an outsider.

`         """
        Prints the person's name and age.

        If the argument 'additional' is passed, then it is appended after the main info.

        Parameters
        ----------
        additional : str, optional
            More info to be displayed (default is None)

        Returns
        -------
        None
        """`

Reply:
i might need to develop a small app first like [this task](https://corvaqa.atlassian.net/browse/DC-675) to add a real hello world example

Reply:
added an example - simplified version of what gamma-depth app does